packages/spotlight/src/container.js

```diff
@@ -951,13 +951,14 @@ function containsContainer (outerContainerId, innerContainerId) {
 function mayActivateContainer (containerId) {
 	const currentContainerId = getLastContainer();
 
-	// If the current container is restricted to 'self-only' and if the next container to be
-	// activated is not inside the currently activated container, the next container should not be
-	// activated.
-	return (
-		!isRestrictedContainer(currentContainerId) ||
-		containsContainer(currentContainerId, containerId)
-	);
+	// If the current container or Its outer containers are restricted to 'self-only' and if the next container to be
```

Comment:
```suggestion
	// If the current container or its outer containers are restricted to 'self-only' and if the next container to be
```

Reply:
The line became longer, so what about refining line breaks?

Reply:
I fixed. Thanks